utils/logger.ts

```diff
@@ -80,7 +80,7 @@ export class Logger {
   // eslint-disable-next-line @typescript-eslint/ban-types
   log(logFn: LogFunction, mergingObject: object, message: string): void {
     if (this.pino === undefined) {
-      throw new Error('Pino is undefined')
+      console.log('Pino is undefined')
```

Comment:
Nit: can we add a bit more context in this message? Something like:

```suggestion
      console.log('Pino is undefined. Application will not log requests until corrected.')
```